rlw.sh

```diff
@@ -242,7 +248,7 @@ printf '%b\n' 'Roblox Linux Wrapper '"$rlwversion"'-'"$branch"
 
 # Don't allow running as root
 if [ "$(id -u)" == "0" ]; then
-   spawndialog error "Roblox Linux Wrapper should not be ran with root permissions."
+   spawndialog error "You're running as root (sudo)!\nThis could mess up Wine so please run this again when you're not in root."
```

Comment:
I think it's obvious that running something with root can mess up Wine. Again, sounds too conversationy to me.